CHANGELOG.md

```diff
@@ -203,6 +203,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Improve messages when integrating Swift pods as static libraries.  
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#7495](https://github.com/CocoaPods/CocoaPods/issues/7495)
+* Prevent duplicated targets from being stripped out from the framework search paths  
```

Comment:
nit press enter and add issue number.

Reply:
Fixed it and moved it to the right place. This does not have a related issue though 😓